tests/acceptance/TestHelpers/UploadHelper.php

```diff
@@ -166,38 +103,7 @@ public static function upload(
 				return $result;
 			}
 		}
-		//finish upload for new chunking
-		if ($chunkingVersion === 2) {
-			$source = $v2ChunksDestination . '/.file';
-			$headers['Destination'] = $baseUrl . "/" .
-				WebDavHelper::getDavPath($davPathVersionToUse, $user) .
-				$destination;
-			$result = WebDavHelper::makeDavRequest(
-				$baseUrl,
-				$user,
-				$password,
-				'MOVE',
-				$source,
-				$headers,
-				null,
-				$xRequestId,
-				null,
-				$davPathVersionToUse,
-				"uploads",
-				null,
-				"basic",
-				false,
-				0,
-				null,
-				[],
-				null,
-				$isGivenStep
-			);
-			if ($result->getStatusCode() >= 400) {
-				return $result;
-			}
-		}
-		self::assertNotNull($result, __METHOD__ . " chunking version $chunkingVersion was requested but no upload was done.");
+		self::assertNotNull($result, __METHOD__ . " chunking version 1 was requested but no upload was done.");
```

Comment:
If the version is bydefault 1, should version 1 be mentioned in Error Message?

Reply:
no need to mention version. you can just write
```
" chunking was requested but no upload was done."
```